studio/components/interfaces/Home/NewProjectPanel/NewProjectPanel.tsx

```diff
@@ -1,28 +1,246 @@
+                    <Link href={`/project/${ref}/storage/buckets`}>
+                      <a>
+                        <Button type="default">Explore Storage</Button>
+                      </a>
+                    </Link>
+                    <Link href="https://supabase.com/docs/guides/storage">
+                      <a>
+                        <Button
+                          className="translate-y-[1px]"
+                          icon={<IconExternalLink size={14} />}
+                          type="default"
+                        >
+                          About Storage
+                        </Button>
+                      </a>
+                    </Link>
+                  </div>
+                </Panel.Content>
+              </Panel>
+
+              <Panel>
+                <Panel.Content className="flex flex-col space-y-4 lg:h-40">
+                  <div className="flex items-center space-x-3">
+                    <div className="bg-scale-600 text-scale-1000 shadow-sm rounded p-1.5">
+                      <IconCode strokeWidth={2} size={16} />
+                    </div>
+                    <h5>Edge Functions</h5>
+                  </div>
+                  <div className="flex flex-grow">
+                    <p className="text-scale-1000 text-sm">
+                      Write custom code without deploying or scaling servers, with fast deploy times
+                      and low latency.
+                    </p>
+                  </div>
+                  <div className="space-x-2 flex items-center">
+                    <Link href={`/project/${ref}/functions`}>
+                      <a>
+                        <Button type="default">Explore Functions</Button>
+                      </a>
+                    </Link>
+                    <Link href="https://supabase.com/docs/guides/functions">
+                      <a>
+                        <Button
+                          className="translate-y-[1px]"
+                          icon={<IconExternalLink size={14} />}
+                          type="default"
+                        >
+                          About Functions
+                        </Button>
+                      </a>
+                    </Link>
+                  </div>
+                </Panel.Content>
+              </Panel>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div className="col-span-4">
+        <div className="space-y-6">
+          <div className="space-y-2">
+            <h3 className="text-xl text-scale-1200">Connecting to your new project</h3>
+            <p className="lg:max-w-sm text-scale-1100">
+              Interact with your database through the{' '}
+              <Link href="https://supabase.com/docs/reference">
+                <a className="text-brand-900">Supabase client libraries</a>
+              </Link>{' '}
+              with your API keys.
+            </p>
+            <p className="lg:max-w-sm text-scale-1100">
```

Comment:
maybe a text-sm for this second paragraph? 

<img width="463" alt="CleanShot 2022-09-23 at 10 04 37@2x" src="https://user-images.githubusercontent.com/105593/191961313-681cdc5a-3b51-44e9-8125-5c19887a3347.png">

Reply:
hmm i'll leave this as the base font size for now for consistency across the text elements on this page specifically